R/angles.R

```diff
@@ -0,0 +1,101 @@
+calculate.angles.radial <- function(v, tree, spread, total.angle) {
+    root.node.id <- v$id[[1]];
+    node.ids <- c(root.node.id);
+
+    total.angle <- total.angle * spread;
+    angles <- numeric(nrow(tree));
+
+    while (length(node.ids) > 0) {
+        current.node.id <- node.ids[1];
+        node.ids <- node.ids[-1];
+
+        parent.id <- tree$parent[tree$tip == current.node.id];
+
+        if (parent.id == -1) {
+            tree$angle[tree$tip == current.node.id] <- 0;
+            }
+
+        child.ids <- tree$tip[tree$parent == current.node.id & !is.na(tree$parent)];
+        num.children <- length(child.ids);
+
+        if (length(child.ids) > 0) {
+            parent.angle <- parent.angle <- tree$angle[tree$tip == current.node.id];
```

Comment:
What's happening in this line? Is this an R construct I don't know (`a <- a <- b`) or is it supposed to be `parent.angle <- parent.angle - ...`?

Reply:
Nope, just a typo.